python/amici/ode_export.py

```diff
@@ -3322,6 +3322,7 @@ def cast_to_sym(value: Union[SupportsFloat, sp.Expr, BooleanAtom],
 
 def smart_subs_dict(sym: sp.Expr,
                     subs: SymbolDef,
+                    reverse: False,
```

Comment:
All usage involves `reverse=True`, so this could be changed to be the default value.